jobmatch/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Seed the Role database with possible user roles
+--
+MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS vals(a,b)
```

Comment:
Spring runs that stuff every boot

On January 25, 2016 7:04:00 PM CST, Charlie Hayes <notifications@github.com> wrote:
>> @@ -0,0 +1,15 @@
>> +--
>> +-- Seed the Role database with possible user roles
>> +--
>> +MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS
>vals(a,b)
>
>Gotcha. For initial seeding, those constraints should never be
>relevant. There's often a command to temporarily turn off constraints
>like that, but I agree we wouldnt want to break the relations.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/idontusenumbers/JobMatch/pull/1/files#r50783173

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.